apps/vehicles/types/next-auth.d.ts

```diff
@@ -1,29 +1,30 @@
-import { JWT } from "next-auth/jwt";
+/* eslint-disable @typescript-eslint/no-unused-vars */
 import NextAuth, { type DefaultSession } from "next-auth";
+import { JWT } from "next-auth/jwt";
 
 declare module "next-auth" {
   interface Session {
     user: User;
   }
 
   interface User {
```

Comment:
**suggestion (non-blocker)**: if both User and JWT interfaces have the same properties (as i can see), consider to create a "common interface" so they can extend from it.

Reply:
absolutely right 🙂 fb71735